packages/dds/merge-tree/src/test/testClient.ts

```diff
@@ -420,6 +422,19 @@ export const createRevertDriver =
                 this.submitOpCallback?.(op);
             },
         insertFromSpec(pos: number, spec: IJSONSegment) {
+            // want the segment's content and the state of insert/remove
+            const test: string[] = [];
+            walkAllChildSegments(client.mergeTree.root,
```

Comment:
Was this walk solely for testing purposes, or should it be removed? @Abe27342

Reply:
should be removed from this method, but if you wanted to put something like it in a `debugDumpTree` method on TestClient I wouldn't strongly object :P. doesn't impact bundle size and useful for diagnosing issues.